concepts/functions/about.md

```diff
@@ -85,13 +110,17 @@ TypeError: add() takes 2 positional arguments but 3 were given
 
 ## Return Value
 
-The return value is a value that is returned to the caller of the function. Return value can be of any data type. It can be used by caller of the function to perform further operations. If the function does not explicitly return a value, the value `None` is returned.
+The return value is a value that is returned to the caller of the function.
+A `return value` can be any data type.
```

Comment:
Wow. Yeah. That is indeed really confusing.  I did not intend at all to delete `It can be used by caller of the function to perform further operations. If the function does not explicitly return a value, the value None is returned.`.  I like that as well.  Let me double check the file....